_routes/conversations.md

```diff
@@ -153,6 +163,12 @@ POST /api/v1/conversations
 }
 ~~~
 
+### Errors
+
+Status code  |  Error reason
+-|-
+422  |  Failed to create conversation due to bad input
```

Comment:
If that's a validation error, the message will contain further details, just like the Rails validation errors.

Reply:
Do I replace it with something like `The conversation creation is not allowed` then?

Reply:
I'd actually write something like `(error message contains information about the failed validations)`, so that it is clear that there is no fixed error message.

Reply:
The table heading here says `Error reason`, so `(error message contains information about the failed validations)` feels contradictory to the heading, because it doesn't name the reason. We can write `failed parameters validation` here as a reason and add `(error message contains information about the failed validations)` too.

Reply:
Also you made a point at IRC before that it's better to have more generic error messages:
```
(21:33:03) DenSchub: [19:34:49] do we need that kind of individual error messaging?
(21:33:03) DenSchub: [19:35:05] can't we just have a "404|Object with provided guid was not found"?
(21:33:03) DenSchub: [19:35:26] or "404|Requested object not found"
(21:33:03) senya: [19:36:40] yeah, probably general message is fine
(21:33:03) DenSchub: [19:37:03] with that, we could have one generic "errors" page and be done with it
```

That's actually why I created a list of messages at the `Errors` page. And here I added error  possible error reasons per-route rather than error messages because error messages are generic.

```
21:33:03) DenSchub: [19:39:56] that increases the amount of duplicate content
(21:33:03) DenSchub: [19:40:07] but if you would prefer that, do it! i don't have a strong opinion
(21:33:03) senya: [19:41:33] It is convenient to have a list of possible errors for every route but if we don't write messages there, then the only thing to place to "errors" section would be status code which looks weird
(21:33:03) DenSchub: [19:42:08] hm. i agree.
(21:33:03) DenSchub: [19:42:19] let's give each route own error documentation, then
```

That's why I wrote it specific for each route here.